src/model/press-package.v1.yaml

```diff
@@ -0,0 +1,43 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Press package
+type: object
+allOf:
+  - $ref: ../snippets/press-package.v1.yaml
+  - properties:
+        content:
+            description: Content
+            type: array
+            items:
+                $ref: ../misc/blocks-all.v1.yaml
+            minItems: 1
+        relatedContent:
+            title: Related content
+            type: array
+            items:
+                oneOf:
+                  - $ref: ../snippets/article-poa.v1.yaml
+                  - $ref: ../snippets/article-vor.v1.yaml
+            minItems: 1
+            uniqueItems: true
+        mediaContacts:
+            title: Media contacts
+            type: array
+            items:
+                allOf:
+                  - $ref: ../misc/person.v1.yaml
+                  - properties:
```

Comment:
@thewilkybarkid how does this look? I will extend the sample fixtures once you are ok with this.